authorization/storage_authorization.go

```diff
@@ -411,16 +643,22 @@ func authorizationsPredicateFn(f influxdb.AuthorizationFilter) kv.CursorPredicat
 	return pred
 }
 
-func filterAuthorizationsFn(filter influxdb.AuthorizationFilter) func(a *influxdb.Authorization) bool {
+func (s *Store) filterAuthorizationsFn(filter influxdb.AuthorizationFilter) func(a *influxdb.Authorization) bool {
 	if filter.ID != nil {
 		return func(a *influxdb.Authorization) bool {
 			return a.ID == *filter.ID
 		}
 	}
 
 	if filter.Token != nil {
+		token := *filter.Token
+		// if AllHashes returns an error, allHashes will be empty and we will ignore a.HashedToken.
+		allHashes, _ := s.hasher.AllHashes(token)
```

Comment:
As above, should we log the error as a debugging clue?

Reply:
Added.